engine/controllers/command_line_parser.cc

```diff
@@ -388,6 +388,12 @@ void CommandLineParser::SetupSystemCommands() {
   update_cmd->group(kSystemGroup);
   update_cmd->add_option("-v", cml_data_.cortex_version, "");
   update_cmd->callback([this] {
+#if !defined(_WIN32)
+    if (getuid()) {
+      CLI_LOG("You are not root, please run the command with `sudo`");
```

Comment:
Maybe a slightly clearer message: 
"Error: Not root user. Please run with sudo."

Reply:
updated, thanks